src_ext/Makefile.sources

```diff
@@ -86,8 +86,8 @@ MD5_stdlib-shims = 09db7af8b4a3a96048a61cb6ae2496ef
 
 $(call PKG_SAME,stdlib-shims)
 
-URL_spdx_licenses = https://github.com/kit-ty-kate/spdx_licenses/releases/download/v1.1.0/spdx_licenses-v1.1.0.tbz
-MD5_spdx_licenses = af8493759aa35b629a324caa3f5ced65
+URL_spdx_licenses = https://github.com/kit-ty-kate/spdx_licenses/releases/download/v1.2.0/spdx_licenses-v1.2.0.tbz
```

Comment:
You've published a .tar.gz for 1.2.0 - does this want changing to reflect that, or will you add a .tbz asset to the release?

Reply:
oh oops. I might not have used ocaml-release-script last time